mlflow/server/js/src/experiment-tracking/components/App.tsx

```diff
@@ -72,7 +73,7 @@ class App extends Component<AppProps> {
         {/* This layer enables intercompatibility between react-router APIs v5 and v6 */}
         {/* TODO: Remove after migrating to react-router v6 */}
         <CompatRouter>
-          <div style={{ height: '100%' }}>
```

Comment:
main problem was that the big div is limited by `height: 100%`, so if the child overflows it (e.g. in the run details pane), we get a white box at the end

Reply:
note: we're going to get rid of App.tsx very soon in favor of [MLflowRouter](https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/MlflowRouter.tsx) which will be enabled after switching [this flag](https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/common/utils/FeatureUtils.ts#L27). 
Most importantly, please refer to general comment for this PR

Reply:
Got it! I'll flip the flag locally and test